index.html

```diff
@@ -1 +1,3 @@
-hello world 
+
```

Comment:
It's best to remove empty spaces at the beginning of files.

When you keep your whitespace tidy and consistent, it's easier to read.

Reply:
I agree. Thank you for the comment.